web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx

```diff
@@ -58,6 +58,7 @@ const FieldUpdateModal: React.FC<Props> = ({
 }) => {
   const t = useT();
   const [form] = Form.useForm();
+  const [buttonDisabled, setButtonDisabled] = useState(false);
```

Comment:
This should be true, right?

Reply:
@KaWaite , `false` because when we open the update modal, the data inside it should already pass the validation.